homeassistant/components/media_player/kodi.py

```diff
@@ -78,22 +81,25 @@
 })
 
 SERVICE_ADD_MEDIA = 'kodi_add_to_playlist'
-SERVICE_SET_SHUFFLE = 'kodi_set_shuffle'
+SERVICE_SET_SHUFFLE = 'kodi_set_shuffle'  # this is the same as mp.shuffle_set
```

Comment:
Let's just eliminate this. It only existed for a short while before core shuffle support was added.